etc/kayobe/environments/ci-multinode/cephadm.yml

```diff
@@ -35,9 +36,17 @@ cephadm_pools:
   - name: vms
     application: rbd
     state: present
+cephadm_pools_manila_cephfs_native:
+  - name: cephfs_data
+    application: cephfs
+    state: present
+  - name: cephfs_metadata
+    application: cephfs
+    state: present
 
 # List of Cephx keys. See stackhpc.cephadm.keys role for format.
-cephadm_keys:
+cephadm_keys: "{{ cephadm_keys_default + (cephadm_keys_manila_cephfs_native if (kolla_enable_manila and kolla_enable_manila_backend_cephfs_native) else [] ) }}"
```

Comment:
Generally best to apply the `bool` filter to any booleans to allow for `no`

Reply:
I've done so on the grouping of the bools `(a and b) | bool` but I've just realised you might have meant do so on each bool individually e.g. `(a | bool and b | bool)`. I can change it round if needed

Reply:
Done